.github/sync-repo-settings.yaml

```diff
@@ -16,6 +16,7 @@ branchProtectionRules:
       - units (11)
       - 'Kokoro - Test: Integration'
       - cla/google
+      - OwlBot Post Processor
```

Comment:
lgtm, but want to hold off a day or two just to make sure we didn't mess anything up with merging in the migration PR :)